cmd/controller/main.go

```diff
@@ -112,15 +136,19 @@ func main() {
 	fasController := fleetautoscalers.NewController(wh, health,
 		kubeClient, extClient, agonesClient, agonesInformerFactory)
 
+	rs := []runner{
+		wh, gsController, gsSetController, fleetController, faController, fasController, metricsController, server,
+	}
+
 	stop := signals.NewStopChannel()
 
 	kubeInformationFactory.Start(stop)
 	agonesInformerFactory.Start(stop)
 
-	rs := []runner{
-		wh, gsController, gsSetController, fleetController, faController, fasController, healthServer{handler: health},
-	}
 	for _, r := range rs {
+		if r == nil {
```

Comment:
Random thought now that I stare at this some more - it _could_ be:

```go
	for _, r := range rs {
		if r != nil {
			go func(rr runner) {
				if runErr := rr.Run(workers, stop); runErr != nil {
					logger.WithError(runErr).Fatalf("could not start runner: %s", reflect.TypeOf(rr))
				}
			}(r)
		}
	}
```
Not sure which is more readable, but figured it might be more worth considering, in case you hadn't.

Reply:
I considered it, I thought my solution created less code indentation, but it seems confusing for you ? Do you want me to rework using your solution ?

Reply:
it took me a second to resolve in my head - but I don't have strong opinions either way. I don't think it's worth blocking on, just wanted to consider it.